### ISSUE TITLE
QA Team- Review Design Wireframe

### ISSUE THREAD

**Issue by lem31** [NoroffFEU/musikkforandrerliv.no#89]

**Comment by lem31** [comment]
- [ ] Visual Hierarchy
- [ ] Layout
- [ ] Consistency
- [ ] Screen size
- [ ] Usability Principles
- [ ] Web Content Accessibility Guidelines (WCAG)
- [ ] Colors
- [ ] Fonts
- [ ] Whitespace and padding
- [ ] Image quality
- [ ] Iconography
- [ ] Micro interactions
- [ ] Error states and feedback
- [ ] Browser compatibility
- [ ] Performance Optimization
- [ ] Design documentation

See Design wireframe/prototype checklist for more info

**Comment by X7-C** [comment]
Computer:

So in terms of coloring:
![image](https://github.com/NoroffFEU/musikkforandrerliv.no/assets/142581465/b61957e6-f21a-46c4-95d0-3bcc4cda878c)
![image](https://github.com/NoroffFEU/musikkforandrerliv.no/assets/142581465/33509267-7349-4e75-9648-12c271d7cafa)
The buttons themselves seem to be okay. It only fails the WCAG AAA. which if possible a slight adjustment could be possibly made? 
Also here with the colors another WCAG AAA that doesnt check out.
![image](https://github.com/NoroffFEU/musikkforandrerliv.no/assets/142581465/23abd8d6-4d2d-45ac-bb02-90de288150a4)
![image](https://github.com/NoroffFEU/musikkforandrerliv.no/assets/142581465/7ab2cf5e-3e14-4542-aa11-fd61edffd97c)

- White space and padding
Very good spacing between different sections of the page. However, on the section "Our Staff" there's a lot of white it can get really bright, and unsure if there will be implemented a dark mode for the page. Will comment further on this.

The bottom Line is a little bit off? Maybe this is on a personal node, however with my OCD I have to mention if it possible to maybe adjust the calendar so that the lines hit directly and not leave a little gap at the bottom?
![image](https://github.com/NoroffFEU/musikkforandrerliv.no/assets/142581465/c5186a70-91b2-4177-ad1a-41b9cecc9eda)

- Image quality
Some Images are lower quality then the others here's hopefully all the ones that I've managed to catch:
![image](https://github.com/NoroffFEU/musikkforandrerliv.no/assets/142581465/454234b6-19d0-47e6-8d59-67f9f9ac5020)
![image](https://github.com/NoroffFEU/musikkforandrerliv.no/assets/142581465/5f63e2ae-577f-45cd-bc2f-032d7ff67edc)

There are 2 others that I am not too sure about because they look a bit of both good and bad quality? Unsure but here they are:
![image](https://github.com/NoroffFEU/musikkforandrerliv.no/assets/142581465/bff661fa-bc10-4a4a-b609-350d4633b4d9)
![image](https://github.com/NoroffFEU/musikkforandrerliv.no/assets/142581465/30361957-9220-497d-a899-c6d4a3096bf0)

- Micro Interactions
Not everything is linked up to each page, but I'm unsure if that was necessary for the Hi-Fi Figma Profile.
Quote me if I am wrong on this, If there will be more to press on or if it Isn't finished yet. 

Fonts personally really enjoy the variety and seeing that the consistency with 3 different fonts is really good.

**Comment by Vickabu** [comment]
I think you guys need to go trough this one again since we are not done with this page after last feedback. Same problem as mention in several issues, lots of comments made mentions components on several pages and therefor needs more time to be completed. 

This comments mention font colours that is no longer used in calender, prototyping like linking sides - that we are not supposed to, Footer is still as mention in other issues in the works - and this footer mention here is allready changed.
